src/app/legacy/components/Promo/index.test.jsx

```diff
@@ -3,10 +3,13 @@ import {
   render,
   screen,
 } from '../../../components/react-testing-library-with-providers';
+import moment from 'moment';
+
 import Promo from '.';
+moment.locale('es');
```

Comment:
Let's move this to within the 11 hours ago test, because it wasn't 100% clear to me how moment was generating the correct translation.

Reply:
moved into the test